bloom-housing/bloom
# feat: MA Cherry Pick sign in copy

# Pull Request Template

## Issue Overview
#3811 #3759 
This PR addresses #issue


## Description

Add Sign In Copy to the Create Account Page
Refactor Sign In Copy Styles
Implement that refactor on sign in page
There are some changes showing up here that should already be in main, but I think once we merge this PR to #3759, that will be resolved.
**Since this is just pulling in MA Copy, which has already been approved, I think this PR only needs one review**

## How Can This Be Tested/Reviewed?
- With MA enabled:
- Go to create account page, test for responsiveness and comparison to Figma
- Ensure you can create an account
- Go to Sign In page and ensure things still look good

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
